locales/en.json

```diff
@@ -36,5 +36,8 @@
     "Footer": "Footer",
     "Pagination": "Pagination",
     "Next post": "Next post",
-    "Previous post": "Previous post"
+    "Previous post": "Previous post",
+    "Sign in": "Sign in",
+    "Sign up": "Sign up",
```

Comment:
Perfect. The `en.json` it the default for new translations. That's why we need them here as well. Thanks! But here's the same issue for `Subscribe now` as in `de.json`.